src/AbstractFlag.php

```diff
@@ -1,29 +1,44 @@
-<?php declare(strict_types = 1);
+<?php declare(strict_types=1);
 
 namespace Aeviiq\Enum;
 
 use Aeviiq\Enum\Exception\InvalidArgumentException;
-use MyCLabs\Enum\Enum;
 
-abstract class AbstractFlag extends Enum
```

Comment:
NOTE that this change is breaking so a major version should be released for this change. If following the symfony guidelines that is: https://symfony.com/doc/current/contributing/code/bc.html#changing-classes (see "Change parent class")